cardinal/server/utils.go

```diff
@@ -183,4 +183,5 @@ func (t *Handler) verifySignatureOfMapRequest(request map[string]interface{}, is
 	} else {
 		return sig.Body, sig, err
 	}
+	return sp.Body, sp, nil
```

Comment:
this is unreachable code - i think we can unwrap the else case here?

Reply:
Srry this isn't fully ready. I'm merging with previous stuff